tests/coverage/extensions.py

```diff
@@ -247,13 +248,15 @@ def __init__(
 
     def add_functions(self, functions, func_type):
         for func in functions.values():
-            self.extensions.add(func["uri"])
+            index = func['uri'].rfind('/extensions')
```

Comment:
fix should not be here. while adding to the map add it correctly, also don't do find of extensions.